docs/3.api/1.composables/use-async-data.md

```diff
@@ -60,6 +61,7 @@ type AsyncDataRequestStatus = 'idle' | 'pending' | 'success' | 'error'
   * _transform_: a function that can be used to alter `handler` function result after resolving
   * _pick_: only pick specified keys in this array from the `handler` function result
   * _watch_: watch reactive sources to auto-refresh
+  * _shallow_: return data in a shallow ref, instead of a deep ref object. This is useful when you want to fine-tune reactivity.
```

Comment:
```suggestion
  * _shallow_: return data in a shallow ref, instead of a deep ref object. This can improve performance when your data does not need to be deeply reactive.
```